system-parachains/people/people-polkadot/src/assets.rs

```diff
@@ -0,0 +1,54 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use super::*;
+
+use frame_support::parameter_types;
+use frame_system::{EnsureNever, EnsureRoot};
+use xcm::latest::prelude::*;
+
+parameter_types! {
+	pub const AssetDeposit: Balance = 0;
```

Comment:
Best if we set something safe here, just in case we allow other (non-root) origins to register assets in the future.

```suggestion
	pub const AssetDeposit: Balance = UNITS;
```